plugin/assets/css/src/customize-controls.css

```diff
@@ -59,18 +59,35 @@
 .customize-control-image-radio-wrap {
 	display: flex;
 	flex-wrap: wrap;
+
+	& h3 {
```

Comment:
Nested css fallback?

Reply:
Same, the compiled file will read `.customize-control-image-radio-wrap h3`